packages/frontend/component/src/ui/input/input.tsx

```diff
@@ -43,22 +44,28 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
     onEnter,
     onKeyDown,
     autoFocus,
+    autoSelect,
     ...otherProps
   }: InputProps,
   upstreamRef: ForwardedRef<HTMLInputElement>
 ) {
-  const handleAutoFocus = useCallback(
+  const handleRef = useCallback(
     (ref: HTMLInputElement | null) => {
       if (ref) {
-        window.setTimeout(() => ref.focus(), 0);
+        if (autoFocus) {
```

Comment:
Can we select even if it is not focused? If not we should judge `autoSelect` too:
```ts
if (autoFocus || autoSelect)
```